packages/backend/src/middlewares/requireAuth.ts

```
@@ -0,0 +1,37 @@
+/**
+ * Middleware to protect routes with JWT authentication
+ * - Checks Authorization header
+ * - Verifies JWT
+ * - Sets req.user if valid
+ * - Returns 401 if missing/invalid
+ */
+const requireAuth = (
+  req: AuthenticatedRequest,
+  res: Response,
+  next: NextFunction
+): Response | void => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)